### ISSUE TITLE
How to send unencoded POST data?

### ISSUE THREAD

**Issue by AlexMercier** [jhy/jsoup#634]
jsoup version: 1.8.3.
Hi. How to send post data without url - encoding?
I mean next, I have post data 

Key | Value
------------ | -------------
{"params":"query | test"}
but when i put this data to Jsoup, this data automaticly encoded special chars:

Key | Value
------------ | -------------
%7B%22params%22%3A%22query | test%22%7D
and serevr dont understend this data.

This is how i use jsoup:
```java
String[] POST_DATA = new String[]{KEY, VALUE, KEY2, VALUE2}; 
Response response = Jsoup.connect(url)
.data(POST_DATA )
.method(Connection.Method.POST)
.execute;
```
Now I'm using android URLCOnnection, because it's can send "raw" data without encoding, but I want to use Jsoup.
this is example how i use now:
`String sParams = String.format("{\"params\":\"%s\"}", "query=test&hitsPerPage=100");`
`{"params":"query=test&hitsPerPage=100"}`
(this is javascript request string params for some site)

**Comment by raphael10241024** [comment]
I hava the same problem ,I what to use "GBK" encode to send the data ,unluckily ,I can't find the setting about encode.Finally i use httpclient insttead.

**Comment by raphael10241024** [comment]
I just read the soure ,I find that there is a method to set the charset infact,Connection.postDataCharset()....

**Comment by jhy** [comment]
Implemented in 1.9.1, you can use Connection.requestBody(string) - http://jsoup.org/apidocs/org/jsoup/Connection.html#requestBody-java.lang.String-
